code/game/objects/items/crab17.dm

```diff
@@ -0,0 +1,226 @@
+	return ..()
+
+/obj/structure/checkoutmachine/proc/start_dumping()
+	accounts_to_rob = SSeconomy.bank_accounts.Copy()
+	accounts_to_rob -= bogdanoff.get_bank_account()
+	for(var/i in accounts_to_rob)
+		var/datum/bank_account/B = i
+		B.dumpeet()
+	dump()	
+
+/obj/structure/checkoutmachine/proc/dump()
+	var/percentage_lost = (rand(1, 10) / 100)
+	for(var/i in accounts_to_rob)
+		var/datum/bank_account/B = i
+		if(!B.being_dumped)
+			continue
+		var/amount = B.account_balance * percentage_lost 
+		bogdanoff.get_bank_account().transfer_money(B, amount)
+		B.bank_card_talk("You have lost [percentage_lost * 100]% of your funds! A spacecoin credit deposit machine is located at: [get_area(src).name].")
+	addtimer(CALLBACK(src, .proc/dump), 150) //Drain every 15 seconds
+
+/obj/structure/checkoutmachine/process()
+	if(!canwalk)
+		return
+	var/anydir = pick(GLOB.cardinals)
+	if(Process_Spacemove(anydir))
+		Move(get_step(src, anydir), anydir)
+
+/obj/structure/checkoutmachine/proc/stop_dumping()
+	for(var/i in accounts_to_rob)
+		var/datum/bank_account/B = i
+		B.being_dumped = FALSE
+
+/obj/effect/dumpeetFall //Falling pod
+	name = ""
+	icon = 'icons/obj/money_machine_64.dmi'
+	pixel_z = 300
+	desc = "Get out of the way!"
+	layer = FLY_LAYER//that wasnt flying, that was falling with style!
+	icon_state = "missile_blur"
+
+/obj/effect/dumpeetTarget
+	name = "Landing Zone Indicator"
+	desc = "A holographic projection designating the landing zone of something. It's probably best to stand back."
+	icon = 'icons/mob/actions/actions_items.dmi'
+	icon_state = "sniper_zoom"
+	layer = PROJECTILE_HIT_THRESHHOLD_LAYER
+	light_range = 2
+	var/obj/effect/dumpeetFall/DF
+	var/obj/structure/checkoutmachine/dump
+	var/mob/living/carbon/human/bogdanoff
+
+/obj/effect/ex_act()
+	return
+
+/obj/effect/dumpeetTarget/Initialize(mapload, user)
+	. = ..()
+	bogdanoff = user
+	addtimer(CALLBACK(src, .proc/startLaunch), 100)
+	sound_to_playing_players('sound/items/dump_it.ogg', 75)
+	deadchat_broadcast("<span class='deadsay'>Protocol CRAB-17 has been activated. A space-coin market has been launched at the station!</span>", turf_target = get_turf(src))
+
+/obj/effect/dumpeetTarget/proc/startLaunch()
+	DF = new /obj/effect/dumpeetFall(drop_location())
+	dump = new /obj/structure/checkoutmachine(null, bogdanoff)
+	priority_announce("The spacecoin bubble has popped! Get to the credit deposit machine at [get_area(src).name] and cash out before you lose all of your funds!", sender_override = "CRAB-17 Protocol")
+	animate(DF, pixel_z = -8, time = 5, , easing = LINEAR_EASING)
+	addtimer(CALLBACK(src, .proc/endLaunch), 5, TIMER_CLIENT_TIME) //Go onto the last step after a very short falling animation
+	addtimer(CALLBACK(src, .proc/playFallingSound), 1, TIMER_CLIENT_TIME) //Go onto the last step after a very short falling animation
+
+/obj/effect/dumpeetTarget/proc/playFallingSound()
+	playsound(src,  'sound/weapons/mortar_whistle.ogg', 80, 1, 6)

```

Comment:
is there really no way to just input playsound directly

Reply:
what did cobby mean by this

Reply:
oops this was a relic from copypasted supplypod code